src/segment-loader.js

```diff
@@ -407,19 +407,32 @@ export const shouldWaitForTimelineChange = ({
   return false;
 };
 
-export const mediaDuration = (audioTimingInfo, videoTimingInfo) => {
-  const audioDuration =
-    audioTimingInfo &&
-    typeof audioTimingInfo.start === 'number' &&
-    typeof audioTimingInfo.end === 'number' ?
-      audioTimingInfo.end - audioTimingInfo.start : 0;
-  const videoDuration =
-    videoTimingInfo &&
-    typeof videoTimingInfo.start === 'number' &&
-    typeof videoTimingInfo.end === 'number' ?
-      videoTimingInfo.end - videoTimingInfo.start : 0;
-
-  return Math.max(audioDuration, videoDuration);
+export const mediaDuration = (timingInfos) => {
+  let maxDuration = 0;
+
+  ['video', 'audio'].forEach(function(type) {
+    const typeTimingInfo = timingInfos[`${type}TimingInfo`];
+
+    if (!typeTimingInfo) {
+      return;
+    }
+    const {start, end} = typeTimingInfo;
+    let duration;
+
+    // eslint-disable-next-line
```

Comment:
is this because our version of eslint doesn't know that `typeof` can return `bigint`?

Reply:
Do you know if later eslint versions support it? If so, we should make a task for us to update eslint so that we can remove this comment (I see it below as well)

Reply:
I updated videojs-standard to support it, I think I can update it as a part of this pull request and remove this line.